deptry/utils.py

```diff
@@ -39,3 +41,64 @@ def load_pyproject_toml(pyproject_toml_path: str = PYPROJECT_TOML_PATH) -> Dict[
             return tomllib.load(pyproject_file)
     except FileNotFoundError:
         raise FileNotFoundError(f"No file `pyproject.toml` found in directory {os.getcwd()}")
+
+
+def find_site_packages_below(root: Path, platform: str) -> Optional[Path]:
+    """Find site packages directory below a virtual environment root"""
+    if platform == "Windows":
+        pattern = "Lib/site-packages"
+    else:
+        pattern = "lib/python*/site-packages"
+    try:
+        return next(root.rglob(pattern))
+    except StopIteration:
+        return None
+
+
+def in_project_virtualenv(
```

Comment:
Maybe we can rename this method to be more clear. Does this detect `deptry_installed_in_virtual_environment` or does this detect `code_is_running_in_virtual_environment`, or something else?